skimage/morphology/_skeletonize_3d_cy.pyx.in

```diff
@@ -0,0 +1,434 @@
+"""
+This is an implementation of the 2D/3D thinning algorithm
+of [Lee94]_ of binary images, based on [IAC15]_. 
+
+The original Java code [IAC15]_ carries the following message:
+
+ * This work is an implementation by Ignacio Arganda-Carreras of the
+ * 3D thinning algorithm from Lee et al. "Building skeleton models via 3-D 
+ * medial surface/axis thinning algorithms. Computer Vision, Graphics, and 
+ * Image Processing, 56(6):462-478, 1994." Based on the ITK version from
+ * Hanno Homann <a href="http://hdl.handle.net/1926/1292"> http://hdl.handle.net/1926/1292</a>
+ * <p>
+ *  More information at Skeletonize3D homepage:
+ *  http://fiji.sc/Skeletonize3D
+ *
+ * @version 1.0 11/13/2015 (unique BSD licensed version for scikit-image)
+ * @author Ignacio Arganda-Carreras (iargandacarreras at gmail.com)
+
+Porting to Cython was done by Evgeni Burovski (evgeny.burovskiy@gmail.com).
```

Comment:
Seconding this. In scikit-image, we keep track of all contributors in CONTRIBUTORS.txt.